adr/ADR-40.md

```diff
@@ -0,0 +1,49 @@
+# Request Many
+
+| Metadata | Value                 |
+|----------|-----------------------|
+| Date     | 2023-06-26            |
+| Author   | @aricart, @scottf     |
+| Status   | Partially Implemented |
+| Tags     | client                |
+
+## Problem Statement
+Have the client support receiving multiple replies from a single request, instead of limiting the client to the first reply.
+
+## Design
+
+Making the request and handling multiple replies is straightforward. Much like a simplified fetch, the developer 
+will provide some basic strategy information that tell how many messages to wait for and how long to wait for those messages.
+
+## Options
+
+#### Count
+
+The number of responses to allow.
+
+* Responses are accepted until the count is reached.
+* Must be used in combination with a time as the count may never be reached.
+
+#### Sentinel
+
+A sentinel is a message with an empty payload. 
+
+* Must be used in combination with a time as a sentinel may never be sent.
+
+#### Max Time
```

Comment:
This is really Max Wait, this is the longest it will wait for additional responses to come in - this value is modified by the "jitter"